futures-cpupool/src/lib.rs

```diff
@@ -430,6 +436,62 @@ impl Builder {
     }
 }
 
+/// Units of work submitted to an `Executor`, currently only created
+/// internally.
+pub struct Run {
```

Comment:
I think it's ok to drop the `pub` here and on the `Run::run` below

Reply:
Done